hdfs-commons/src/main/java/org/apache/mesos/hdfs/util/FailureUtils.java

```diff
@@ -0,0 +1,19 @@
+package org.apache.mesos.hdfs.util;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+/**
+ * Failure utilities.
+ */
+public class FailureUtils {
+  private static final Log log = LogFactory.getLog(FailureUtils.class);
+
```

Comment:
I kind of like the warning.  I don't think we should be calling System.exit and taking the whole JVM down all over the place.  The warning prodded me into centralizing a location for the exit and annotating why I'm doing something out of the ordinary.  The warning indicates that throwing RuntimeExceptions is preferred, but I didn't want to deal with tracking all possible threads etc.  I wanted fail fast behavior, so that's documented very clearly now.  I don't know why the Executor's are getting way with this warning.  Maybe they have different build configuration.  If anything I'd prefer their configuration to generate the warning.